src/input/tests/simulation/bad/mc/move_acceptance-3.toml

```diff
@@ -0,0 +1,17 @@
+[input]
+version = 1
+
+[[systems]]
+file = "../../CO2.xyz"
+
+[[simulations]]
+nsteps = 1
+
+[simulations.propagator]
+type = "MonteCarlo"
+temperature = "300 K"
+update_frequency = -1
+#^ 'update_frequency' must be a positive integer in Monte-Carlo propagator
+moves = [
+    {type = "Rotate", delta = "6 A", frequency = 0.3, target_acceptance = 0.5}
```

Comment:
Sure. Currently, this is not checked within `input` but in `core`. It will panic with a hint to the mcmove source file. Should we catch this in the input directly?